src/models/community_business.ts

```diff
@@ -187,6 +187,18 @@ export const CommunityBusinesses: CommunityBusinessCollection = {
     return res || null;
   },
 
+  async getTemporary (client) {
+    return client('organisation')
+    .select({
+      id: 'organisation.organisation_id',
+      name: 'organisation.organisation_name',
+      createdAt: 'organisation.created_at',
+      modifiedAt: 'organisation.modified_at',
+      deletedAt: 'organisation.deleted_at',
+    })
+    .where('organisation.organisation_name', 'like', 'TEMPORARY ACCOUNT: %');
```

Comment:
As soon as a temp account is created, I can log in as it into the visitor app, rename the organisation to anything that doesn't include `TEMPORARY ACCOUNT`, and suddenly it's a fully fledged organisation.

This is why I wanted things to be [represented directly](https://github.com/TwinePlatform/twine-api/pull/349#discussion_r260258368), because encoding things indirectly like this leads to unforseen issues.

We'll need to address #361 in this PR.

Reply:
😬 I had not considered this!